plugins/woocommerce/client/blocks/tests/e2e/utils/frontend/frontend-utils.page.ts

```diff
@@ -41,27 +41,31 @@ export class FrontendUtils {
 				url.includes( 'batch' )
 			);
 		} );
+		// Wait for all XHR requests to finish. Some blocks do another request to the cart endpoint after the above.
+		// eslint-disable-next-line playwright/no-networkidle
+		await this.page.waitForLoadState( 'networkidle' );
 	}
 
 	async goToCheckout() {
-		await this.page.goto( '/checkout' );
+		await this.page.goto( this.requestUtils.baseURL + '/checkout' );
+		// Wait for the email field to be visible
 		await this.page.locator( '#email' ).waitFor();
 	}
 
 	async goToCart() {
-		await this.page.goto( '/cart' );
+		await this.page.goto( this.requestUtils.baseURL + '/cart' );
```

Comment:
The `baseUrl` is automatically added by the Playwright, no need to add it explicitly.